src/Bindings/LuaChunkStay.h

```diff
@@ -32,7 +32,7 @@ class cLuaChunkStay
 public:
 	cLuaChunkStay();
 
-	~cLuaChunkStay() { }
+	~cLuaChunkStay() override { }
```

Comment:
I think an `override` should always be a `virtual` (it is implied, but it makes sense to make it explicit)